javascript/javascript1/week3/note-taking-app/note-taking-app.js

```diff
@@ -0,0 +1,30 @@
+const notes = [];
+
+function saveNote(content, id) {
+  notes.push({
+    content: content,
+    id: id,
+  });
```

Comment:
You can use object shorthand notation when the property name and the variable name holding the value are the same. It would look like this:
```suggestion
  notes.push({ content, id });
```